docs/en/siem/siem-ui.asciidoc

```diff
@@ -2,34 +2,78 @@
 [role="xpack"]
 == SIEM UI
 
-The {siem-app} is a highly interactive workspace designed for security 
-analysts. It provides a clear, detailed overview of events from your 
-environment, and you can use the interactive UI to drill down into areas of 
+The {siem-app} is a highly interactive workspace designed for security
+analysts. It provides a clear, detailed overview of events from your
+environment, and you can use the interactive UI to drill down into areas of
 interest.
 
 The *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* bar is available
 throughout the {siem-app} for searching and filtering.
 
-NOTE: The default index glob patterns defined for {siem-soln} events are 
+NOTE: The default index glob patterns defined for {siem-soln} events are
 `endgame-*`, `auditbeat-*`, `winlogbeat-*`, `filebeat-*`, `packetbeat-*`,
-and `apm-*-transaction*`. You can change the default glob patterns in {kib} -> 
+and `apm-*-transaction*`. You can change the default glob patterns in {kib} ->
 Management -> Advanced Settings -> `siem:defaultIndex`.
 
 
 [float]
 [[siem-overview-ui]]
 === Overview
 
-The Overview page provides a high-level view into security events available
+The Overview page provides a high-level view of security events available
 for analysis, and can help surface problems with data ingestion.
 
-Notice the *Timeline* on the right of the {siem-ui}. It is always available for
-starting an investigation, whether in its collapsed or expanded state. Identify
-items you want to dig into, and drag and drop them to the Timeline. Those items
-are waiting for you when you are ready to start your investigation.
+All histograms and graphs on the overview page contain an **Inspect** button. Use inspect to receive details on count statistics, and to copy the JSON request and response set by the SIEM UI.
```

Comment:
Optional:

The Inspect button affords users "a peek behind the curtain", so they may better understand the data being surfaced by histograms on the overview, and throughout the app.

That said, users who may be less familiar with the Elasticsearch query DSL shown by the Inspect button don't need to understand the queries right away in order to become productive with the SIEM app.

Consider placing the new `All histograms and graphs...` paragraph in a callout that indicates this is a "tip", as opposed to a requirement for effectively using the app.